src/aics-image-viewer/components/shared/SmarterSlider.tsx

```diff
@@ -4,23 +4,23 @@ import Nouislider, { NouisliderProps } from "nouislider-react";
 type CallbackArgs = Parameters<NonNullable<NouisliderProps["onStart"]>>;
 
 const MemoedNouislider = React.memo(
-  Nouislider as React.ComponentType<NouisliderProps & { shouldUpdate: boolean }>,
-  ({ shouldUpdate }) => shouldUpdate
+  Nouislider as React.ComponentType<NouisliderProps & { noUpdate: boolean }>,
+  ({ noUpdate }) => noUpdate
 );
 
 /** A wrapper around `Nouislider` that prevents updates while the slider is being dragged. */
 const SmarterSlider: React.FC<NouisliderProps> = (props) => {
-  const [shouldUpdate, setShouldUpdate] = React.useState(true);
+  const [noUpdate, setNoUpdate] = React.useState(false);
   const wrapEventHandler = (shouldUpdate: boolean, handler?: (...args: CallbackArgs) => void) => {
     return (...args: CallbackArgs) => {
-      setShouldUpdate(shouldUpdate);
+      setNoUpdate(shouldUpdate);
```

Comment:
should the argument here be called `shouldNotUpdate` then?  As in `setNoUpdate(shouldNotUpdate);` ?

Reply:
Knew I was gonna miss something